examples/_test_progressive_loading.py

```diff
@@ -33,14 +50,62 @@ def test_add_progressive_loading_image_zoom_in(mandelbrot_arrays):
     viewer = napari.Viewer()
     viewer.camera.zoom = 0.0001
     add_progressive_loading_image(mandelbrot_arrays, viewer=viewer)
-    viewer.camera.zoom = 0.001  # only fails if we change visible scales
-
+    viewer.camera.zoom = 0.001
+    
 
 def test_add_progressive_loading_image_zoom_out(mandelbrot_arrays):
     viewer = napari.Viewer()
     viewer.camera.zoom = 0.001
     add_progressive_loading_image(mandelbrot_arrays, viewer=viewer)
-    viewer.camera.zoom = 0.0001  # only fails if we change visible scales
+    viewer.camera.zoom = 0.0001
+
+# TODO test for nothing visible on initial load and for nothing visible after camera movement
+def test_add_progressive_loading_image_no_visible(mandelbrot_arrays):
+    viewer = napari.Viewer()
+    non_visible_center = (0.0, -3242614, -9247091)
+    start_zoom = 0.00005
+
+    add_progressive_loading_image(mandelbrot_arrays, viewer=viewer)
+    viewer.camera.zoom = start_zoom
+    viewer.camera.center = non_visible_center
+
+
+def test_add_progressive_loading_image_zoom_before_load(mandelbrot_arrays):
+    # TODO fails for max_levels=8
+    viewer = napari.Viewer()
+    start_center = (0.0, 4194303.5, 4194303.5)
+    non_visible_center = (0.0, -3242614, -9247091)
+    start_zoom = 0.00005
+    viewer.camera.zoom = start_zoom
+    viewer.camera.center = non_visible_center
+    add_progressive_loading_image(mandelbrot_arrays, viewer=viewer)
+    
+def test_MultiScaleVirtualData_set_interval(mandelbrot_arrays):
+    # viewer = napari.Viewer()
+    # multi_data = _progressive_loading.MultiScaleVirtualData(mandelbrot_arrays)
+    # # add_progressive_loading_image(mandelbrot_arrays, viewer=viewer)
+    
+    # # prev_max_coord = multi_data._max_coord
+    # # prev_min_coord = multi_data._min_coord
+
+    # # pan to reset coords
+    # coords = tuple([
+    #     slice(0, 1024, None),
+    #     slice(0, 1024, None)
+    # ])
+    # # coords = tuple([
+    # #     slice(0, 673, None),
+    # #     slice(0, 0, None)
+    # # ])
+    # # coords = tuple([
+    # #     slice(0, 1346, None),
+    # #     slice(0, 0, None)
+    # # ])
```

Comment:
Still working on this as a test case. These are in the wrong coordinates. They should be in canvas coordinates, but I couldn't see how to convert without rendering something.